styles/gallery.css

```diff
@@ -0,0 +1,135 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  body{
+    height: 100vh;
+    overflow-x: hidden;
+    background-image: url('../images/gallery_page_background.jpg');
+    background-size: cover;
+    background-repeat: repeat;
+}
+#gallery_heading{
+  color: #6c63ff;
+}
+.center {
+  width: 100vw;
```

Comment:
since div is a block-level element, shoudn't it take 100% width (100vw) by default?